FloconDesktop/composeApp/src/commonMain/kotlin/io/github/openflocon/flocondesktop/messages/domain/model/DeviceIdAndPackageNameDomainModel.kt

```diff
@@ -10,7 +10,7 @@ data class DeviceIdAndPackageNameDomainModel(
 
 // TODO Move
 
-fun DeviceIdAndPackageNameDomainModel.toRemote() = FloconDeviceIdAndPackageName(
+fun DeviceIdAndPackageNameDomainModel.toRemote() = FloconDeviceIdAndPackageNameDataModel(
```

Comment:
the toRemove should be placed in the data module, domain module is not aware of data models